### PR TITLE
feat(RDS): import RDS resource, unit test and document.

### PR DESCRIPTION
**Pull request by Zippo-Wang**
**What this PR does / why we need it**:
import RDSresource, unit test and document.

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
### resource_flexibleengine_rds_backup

=== RUN   TestAccBackup_mysql_basic
=== PAUSE TestAccBackup_mysql_basic
=== CONT  TestAccBackup_mysql_basic
--- PASS: TestAccBackup_mysql_basic (1185.01s)
PASS

coverage: 2.1% of statements in ../../../terraform-provider-flexibleengine/...


=== RUN   TestAccBackup_sqlserver_basic
=== PAUSE TestAccBackup_sqlserver_basic
=== CONT  TestAccBackup_sqlserver_basic
--- PASS: TestAccBackup_sqlserver_basic (1931.04s)
PASS

coverage: 1.9% of statements in ../../../terraform-provider-flexibleengine/...


=== RUN   TestAccBackup_pg_basic
=== PAUSE TestAccBackup_pg_basic
=== CONT  TestAccBackup_pg_basic
--- PASS: TestAccBackup_pg_basic (1034.23s)
PASS

coverage: 1.9% of statements in ../../../terraform-provider-flexibleengine/...
```

```
### data_source_flexibleengine_rds_backup

=== RUN   TestAccDatasourceBackup_basic
=== PAUSE TestAccDatasourceBackup_basic
=== CONT  TestAccDatasourceBackup_basic
--- PASS: TestAccDatasourceBackup_basic (1108.51s)
PASS

coverage: 1.9% of statements in ../../../terraform-provider-flexibleengine/...


=== RUN   TestAccDatasourceBackup_auto_basic
=== PAUSE TestAccDatasourceBackup_auto_basic
=== CONT  TestAccDatasourceBackup_auto_basic
--- PASS: TestAccDatasourceBackup_auto_basic (412.16s)
PASS

coverage: 1.9% of statements in ../../../terraform-provider-flexibleengine/...

=== RUN   TestAccDatasourceBackup_incremental_basic
=== PAUSE TestAccDatasourceBackup_incremental_basic
=== CONT  TestAccDatasourceBackup_incremental_basic
--- PASS: TestAccDatasourceBackup_incremental_basic (432.17s)
PASS

coverage: 1.7% of statements in ../../../terraform-provider-flexibleengine/...
```

```
### data_source_flexibleengine_engine_versions

=== RUN   TestAccRdsEngineVersionsV3DataSource_basic
=== PAUSE TestAccRdsEngineVersionsV3DataSource_basic
=== CONT  TestAccRdsEngineVersionsV3DataSource_basic
--- PASS: TestAccRdsEngineVersionsV3DataSource_basic (44.08s)
PASS

coverage: 1.7% of statements in ../../../terraform-provider-flexibleengine/...

=== RUN   TestAccRdsEngineVersionsV3DataSource_PostgreSQL_basic
=== PAUSE TestAccRdsEngineVersionsV3DataSource_PostgreSQL_basic
=== CONT  TestAccRdsEngineVersionsV3DataSource_PostgreSQL_basic
--- PASS: TestAccRdsEngineVersionsV3DataSource_PostgreSQL_basic (38.72s)
PASS
coverage: 1.7% of statements in ../../../terraform-provider-flexibleengine/...

=== RUN   TestAccRdsEngineVersionsV3DataSource_SQLServer_basic
=== PAUSE TestAccRdsEngineVersionsV3DataSource_SQLServer_basic
=== CONT  TestAccRdsEngineVersionsV3DataSource_SQLServer_basic
--- PASS: TestAccRdsEngineVersionsV3DataSource_SQLServer_basic (43.08s)
PASS

coverage: 1.7% of statements in ../../../terraform-provider-flexibleengine/...
```

```
### data_source_flexibleengine_storage_types

=== RUN   TestAccDatasourceStoragetype_basic
=== PAUSE TestAccDatasourceStoragetype_basic
=== CONT  TestAccDatasourceStoragetype_basic
--- PASS: TestAccDatasourceStoragetype_basic (41.38s)
PASS

coverage: 1.7% of statements in ../../../terraform-provider-flexibleengine/...

=== RUN   TestAccDatasourceStoragetype_PostgreSQL_basic
=== PAUSE TestAccDatasourceStoragetype_PostgreSQL_basic
=== CONT  TestAccDatasourceStoragetype_PostgreSQL_basic
--- PASS: TestAccDatasourceStoragetype_PostgreSQL_basic (35.54s)
PASS

coverage: 1.7% of statements in ../../../terraform-provider-flexibleengine/...

=== RUN   TestAccDatasourceStoragetype_SQLServer_basic
=== PAUSE TestAccDatasourceStoragetype_SQLServer_basic
=== CONT  TestAccDatasourceStoragetype_SQLServer_basic
--- PASS: TestAccDatasourceStoragetype_SQLServer_basic (36.72s)
PASS
coverage: 1.7% of statements in ../../../terraform-provider-flexibleengine/...
```

```
### data_source_flexibleengine_rds_instances
=== RUN   TestAccRdsInstanceDataSource_basic
=== PAUSE TestAccRdsInstanceDataSource_basic
=== CONT  TestAccRdsInstanceDataSource_basic
--- PASS: TestAccRdsInstanceDataSource_basic (1086.39s)
PASS

coverage: 1.9% of statements in ../../../terraform-provider-flexibleengine/...


=== RUN   TestAccRdsInstanceDataSource_ha_basic
=== PAUSE TestAccRdsInstanceDataSource_ha_basic
=== CONT  TestAccRdsInstanceDataSource_ha_basic
--- PASS: TestAccRdsInstanceDataSource_ha_basic (165.64s)
PASS

=== RUN   TestAccRdsInstanceDataSource_replica_basic
=== PAUSE TestAccRdsInstanceDataSource_replica_basic
=== CONT  TestAccRdsInstanceDataSource_replica_basic
--- PASS: TestAccRdsInstanceDataSource_replica_basic (294.93s)
PASS

=== RUN   TestAccRdsInstanceDataSource_enterprise_basic
=== PAUSE TestAccRdsInstanceDataSource_enterprise_basic
=== CONT  TestAccRdsInstanceDataSource_enterprise_basic
--- PASS: TestAccRdsInstanceDataSource_enterprise_basic (33.76s)
PASS

```

```
### resource_flexibleengine_rds_sql_audit
=== RUN   TestAccSQLAudit_basic
=== PAUSE TestAccSQLAudit_basic
=== CONT  TestAccSQLAudit_basic
--- PASS: TestAccSQLAudit_basic (1077.14s)
PASS

coverage: 1.9% of statements in ../../../terraform-provider-flexibleengine/...
```

```
### resource_flexibleengine_rds_cross_region_backup_strategy
=== RUN   TestAccBackupStrategy_basic
=== PAUSE TestAccBackupStrategy_basic
=== CONT  TestAccBackupStrategy_basic
--- PASS: TestAccBackupStrategy_basic (1027.76s)
PASS

coverage: 1.9% of statements in ../../../terraform-provider-flexibleengine/...
```

### CODE REVIEW

File: docs/data-sources/rds_storage_types.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+* `instance_mode` - (Optional, String) Specifies the HA mode. The value options are as
+  follows: **single**, **ha**, **replica**.
+
+## Attribute Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The resource ID.
+
+* `storage_types` - Indicates the DB instance specifications information list. For details, see Data structure of
```
[the remark this answers is not in the document]
Comment (Zippo-Wang) [reply docs/data-sources/rds_storage_types.md:42]: the field name in document is consistent with schema definition. It is `storage_types`.

File: docs/resources/rds_backup.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+* `instance_id` - (Required, String, ForceNew) Specifies the instance id.
+  Changing this parameter will create a new resource.
+
+* `description` - (Optional, String, ForceNew) The description about the backup.  
+  It contains a maximum of 256 characters and cannot contain the following special characters: **> ! < " & ' =**.
+  Changing this parameter will create a new resource.
+
+* `databases` - (Optional, List, ForceNew) List of self-built Microsoft SQL Server databases that are partially
+  backed up. (Only Microsoft SQL Server supports partial backups). The [databases](#rds_databases) structure is
+  documented below. Changing this parameter will create a new resource.
```
[the remark this answers is not in the document]
Comment (Zippo-Wang) [reply docs/resources/rds_backup.md:42]: It is not mandatory, the `Changing ... new resource` should be declared when the field is `ForceNew`. As for it's location, one way is added behind the field description. Another way is starting a new line.

File: docs/resources/rds_cross_region_backup_strategy.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+  destination_project_id = var.destination_project_id
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the resource.
+  If omitted, the provider-level region will be used. Changing this parameter will create a new resource.
```
[the remark this answers is not in the document]
Comment (Zippo-Wang) [reply docs/resources/rds_cross_region_backup_strategy.md:30]: ditto

File: docs/resources/rds_cross_region_backup_strategy.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+* `instance_id` - (Required, String, ForceNew) Specifies the ID of the RDS instance.
+
```
[the remark this answers is not in the document]
Comment (Zippo-Wang) [reply docs/resources/rds_cross_region_backup_strategy.md:33]: ditto

File: docs/data-sources/rds_storage_types.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Zippo-Wang) [reply docs/data-sources/rds_storage_types.md:42]: fix

**Review by Zhukun-Huawei — COMMENTED** [review]
/lgtm

**Pull request merged**